### PR TITLE
Update configs from LAMB to AdamW

### PR DESCRIPTION
**Pull request by redoctopus**
Signed-off-by: Jocelyn Huang <jocelynh@nvidia.com>

# What does this PR do ?

Replaces LAMB optimizer (unstable, hangs during multi-GPU training) with AdamW for FastPitch and Mixer-TTS configs.

**Collection**: TTS

# Changelog 
- Replace LAMB defaults with AdamW
- Update `fastpitch.py` example script to use existing config

**PR Type**:
- [ ] New Feature
- [x] Bugfix
- [ ] Documentation

### CODE REVIEW

File: examples/tts/fastpitch.py [hunk 1]
```diff
@@ -21,7 +21,7 @@
 
 
 # TODO(Oktai15): remove this script in 1.8.0 version
```
Comment (subhankar-ghosh) [inline examples/tts/fastpitch.py:23]: Can you please remove this TODO(...) line.

**Comment by subhankar-ghosh** [comment]
Can you please replace LAMB with ADAMW in this config as well https://github.com/NVIDIA/NeMo/blob/main/examples/tts/conf/fastpitch_align_44100.yaml#L213

**Comment by redoctopus** [comment]
> Can you please replace LAMB with ADAMW in this config as well https://github.com/NVIDIA/NeMo/blob/main/examples/tts/conf/fastpitch_align_44100.yaml#L213

Sure thing, let me close this PR and open a new one based off of main.

**Comment by redoctopus** [comment]
See #4233

File: examples/tts/conf/fastpitch_align_v1.05.yaml [hunk 2]
```diff
@@ -210,9 +210,9 @@ model:
     n_layers: 2
 
   optim:
-    name: lamb
+    name: adamw
     lr: 1e-1
-    betas: [0.9, 0.98]
```
Comment (XuesongYang) [inline examples/tts/conf/fastpitch_align_v1.05.yaml:215]: nit: could you please advise me any resource showing 0.999 is better than 0.98? sorry, I lack the experience of adamW.

**Pull request closed without merging**